include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2176,21 +2182,21 @@ uint8_t bt_le_per_adv_sync_get_index(struct bt_le_per_adv_sync *per_adv_sync);
  *
  * This function is to get the periodic advertising sync object from
  * the array index.
- * The array has CONFIG_BT_PER_ADV_SYNC_MAX elements.
+ * The array has @kconfig{CONFIG_BT_PER_ADV_SYNC_MAX} elements.
  *
  * @param index The index of the periodic advertising sync object.
- *              The range of the index value is 0..CONFIG_BT_PER_ADV_SYNC_MAX-1
+ *              The range of the index value is 0..@kconfig{CONFIG_BT_PER_ADV_SYNC_MAX}-1
  *
  * @return The periodic advertising sync object of the array index or NULL if invalid index.
  */
 struct bt_le_per_adv_sync *bt_le_per_adv_sync_lookup_index(uint8_t index);
 
-/** @brief Advertising set info structure. */
+/** @brief Periodic advertising set info structure. */
 struct bt_le_per_adv_sync_info {
 	/** Periodic Advertiser Address */
 	bt_addr_le_t addr;
 
-	/** Advertiser SID */
+	/** Advertising Set Identifier. */
 	uint8_t sid;
 
 	/** Periodic advertising interval (N * 1.25 ms) */
```

Comment:
> /** Advertiser PHY */

How to set it? Could any random number be inserted here? Any @ref to defined constants?

Reply:
Added BT_GAP_SID_MIN :)

Reply:
My comment was about "Advertiser PHY" which is few lines below. I always stumble upon this and we get questions from customers on how to setup the desired PHY, and then you have to spend five minutes on searching options to use. Basically, please point to `@ref bt_gap_le_phy`.